mmv1/openapi_generate/parser.go

```diff
@@ -451,7 +451,12 @@ func buildProperties(props openapi3.Schemas, required []string) []*api.Type {
 
 // Trims whitespace from the ends of lines in a description to force multiline
 // formatting for strings with newlines present
-func trimSpacesFromDescription(description string) string {
+// Also trim "Output only." and "Required." from descriptions as this gets duplicated
+func trimDescription(description string) string {
+	description, _ = strings.CutPrefix(description, "Optional. ")
+	description, _ = strings.CutPrefix(description, "Output only. ")
```

Comment:
Possibly also `[Output Only]` and `[Output only]`?

Reply:
Have we seen that? That doesn't look like the standard form that gets generated

Reply:
Ah, developer connect. It looks like they manually wrote [Output only] in their comment that gets copied downstream. I'd lean towards not including that as it's a one-off manual addition

Reply:
okay sgtm